codalab/worker/worker.py

```diff
@@ -89,6 +90,7 @@ def __init__(
         self.shared_file_system = shared_file_system
 
         self.exit_when_idle = exit_when_idle
+        self.exit_number_jobs = exit_number_jobs
```

Comment:
I think it's clearer / safer to leave this as an immutable number and have another value `num_runs` which counts up.

Reply:
Updated.